### ISSUE TITLE
SCS2 with ROS2 

### ISSUE THREAD

**Issue by MishaSavchenko** [ihmcrobotics/valkyrie#4]
I am able to set up and run the SCS2 and these are my modules that I am selecting at launch.
![image](https://github.com/ihmcrobotics/valkyrie/assets/17458981/5ab28ed3-d55d-4e3f-a9a7-e2d170539939)
I can see the topics published out of the sim in ROS 1, I cannot do the same thing with ROS 2.
Is there a way to communicate with SCS via ROS2 interfaces?

**Comment by SylvainBertrand** [comment]
ROS2 starts automatically as soon as you have NetworkProcessor checked. Maybe the domain ID is not set properly. Not sure how you set it in ROS2, for the IHMC software create the file `~/.ihmc/IHMCNetworkParameters.ini` with the following:
```
RTPSDomainID=15
```
Where you replace "15" with your domain ID.
Lemme know if that helps, I'll need to add that to the readme.

**Comment by MishaSavchenko** [comment]
Yeah that clarifies that parameter! Once i set the domain id I was able to get the ROS2 interface.
I saw that IHMCNetworkParameters.ini file is being loaded, and if it cant be found the RTPSDomainID is set randomly, I believe.
Is there an example of the IHMCNetworkParameters.ini with all the parameters that could be set, I wasn't able to find it in the ihmc-open-robotics-software repo or this one.
